lib/remote_ip/block.ex

```diff
@@ -1,5 +1,5 @@
 defmodule RemoteIp.Block do
-  use Bitwise
+  import Bitwise
```

Comment:
👍 Looks like `use Bitwise` was simply deprecated circa 1.14: https://github.com/elixir-lang/elixir/blob/v1.14/CHANGELOG.md#4-hard-deprecations

According to the changelog, `~~~` was also deprecated in favor of `bnot`. However, that update hasn't been made below (on lines 79 & 85). It apparently doesn't generate errors (possibly because I have `~~~` inside of bitstrings?), but it should be updated regardless.